forge-gui/res/languages/zh-CN.properties

```diff
@@ -3287,3 +3287,4 @@ lblWurm=亚龙
 lblYeti=雪怪
 lblZombie=灵俑
 lblZubera=无面鬼
+lblWouldYouLikeToPlaneswalk=Would you like to planeswalk?
```

Comment:
Something like this would be great for all languages
![image](https://github.com/Card-Forge/forge/assets/103371817/3b0d314b-b618-4d71-990a-5f333a11c7b1)

Reply:
You can see most effects are lumped together and alphabetical